tasks/lib/run.js

```diff
@@ -104,9 +105,79 @@ RunUtil.displayScriptOutput = function(grunt, outputs, data) {
   }
 };
 
+function getConstructorName(value) {
+  if(value == null) { // is it undefined or null
+    return String(value);
+  }
+
+  return value.constructor.name;
+}
+
+function getFallbackErrorMessage(err) {
+  switch(err.__fxotype__) {
+    case 'AuthError':
+      return 'The request failed because of an authorization problem.';
+    case 'ServiceError':
+      return 'There was an error with your task, please contact support.';
+    case 'WebhookIgnoreError':
+      return 'The data received by the webhook was ignored.';
```

Comment:
Yes, I could just make it so that `WebhookIgnoreError`s will be silently ignored.
I think that core will handle this type of error as a retryable error though? I haven't had a chance to verify that yet.